lib/pg-log/pglite-adapter.ts

```
@@ -0,0 +1,41 @@
+      const prepared = values.map(prepareValue);
+      const result = await db.query(text, prepared);
+      return {
+        rows: result.rows,
+        rowCount: result.affectedRows ?? result.rows.length,
+        command: undefined,
+      };
+    },
+  };
+
+  return instance;
+}
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)